### DISCUSSION TITLE
----  Please Add JITTER  -----

### DISCUSSION THREAD

**Discussion by Persianfriend** [MortezaBashsiz/CFScanner#557] [Ideas]
سلام و خسته نباشید،

میخواستم خواهش کنم در صورتی که صلاح میدونید حالا که انتخاب آیپی بر اساس سرعت آپلود رو اضافه کردید انتخاب آیپی بر اساس JITTER رو هم اضافه کنید، فکر میکنم آیپی تمیز رابطه مستقیم با JITTER داره . به این شکل که مثلا آیپیهایی که آپلود 10KB  دارند و JITTER آنها زیر 50 است.

درود و سپاس از زحمات شما برای دسترسی به اینترنت آزاد

**Comment by Persianfriend** [comment]
درود مجدد،
جیتر در تست سرعت پینگ یعنی میانگین اندازه اختلاف سرعت تاخیر
پینگها در در تست پینگ برحسب میلی ثانیه به این معنی که وقتی مثلا پنج بار از یک آیپی پینگ
میگیریم اگر میانگین اختلاف مقدار پینگها  خیلی کم باشه اونوقت مقدار جیتر هم خیلی
کوچک خواهد بود مثلا 8 میلی ثانیه ولی اگر اختلاف پینگهایی که طی اون پنج بار گرفتیم
خیلی زیاد باشه مثلا یکبار 120 باشه، یکبار 800، یکبار 1220، یکبار560 و یکبار
هم 980  اونوقت طبیعتا جیتر هم عدد بزرگی میشه که در این مثال جیتر میشه  275 میلی ثانیه
 البته شما خودتون استاد هستید و صاحبنظر و ممکنه این برداشت من اشتباه باشه
ولی طبق تجربه با اسکنر ویندوز آیپیهایی که اسکنر پیدا میکرد فقط اونهایی که
جیتر کوچکتر داشتند قابلیت آپلود با سرعت مناسب داشتند و آنهایی که جیتر بالای
200 و 300 داشتند ممکن بود حتی سرعت دانلود بالاتری داشته باشند ولی سرعت
آپلودشان نزدیک به صفر بود.
در نهایت به این نتیجه رسیدم که وقتی جیتر بالاست نشان دهنده اختلال
روی آیپی است واون آیپی، آپلودی نزدیک به صفر خواهد داشت که البته خودم شخصا
اینو تجربه کردم ولی نمیدونم آیا در همه شرایط اینجور هست یا نه برای همین
مطرح کردم که اگر امکان داره شما که دانش و ابزار این کار رو دارید بررسی فرمایید
🙏

**Comment by rayan6x** [comment]
🙏🙏🙏🙏🙏🤲🤲🤲🤲🤲🤲🤲🤲🤲🤲🌷🌷🌷🌷🌷🌷🌷🌷🤭🤭🤭🤭🤭🤭🤭🤭🤭🤭🤭🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏
